FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/ClientConfigurationViewModel.kt

```diff
@@ -29,7 +29,7 @@ class ClientConfigurationViewModel(application: Application) : BaseAndroidViewMo
   val clientConfigurationLiveData: MutableLiveData<Result<ClientConfigurationResponse>> =
     MutableLiveData(Result.none())
 
-  fun fetchClientConfiguration(idToken: String, fesUrl: String? = null) {
+  fun fetchClientConfiguration(idToken: String, customFesUrl: String) {
```

Comment:
What does this `customFesUrl` value here contain exactly?

Reply:
Outdated